apis/v1alpha1/types.go

```diff
@@ -105,9 +235,88 @@ type PutPartnerEventsRequestEntry struct {
 	Source     *string `json:"source,omitempty"`
 }
 
+// Represents a target that failed to be added to a rule.
+type PutTargetsResultEntry struct {
+	TargetID *string `json:"targetID,omitempty"`
+}
+
+// These are custom parameters to be used when the target is a Amazon Redshift
+// cluster to invoke the Amazon Redshift Data API ExecuteStatement based on
+// EventBridge events.
+type RedshiftDataParameters struct {
+	Database         *string `json:"database,omitempty"`
+	DBUser           *string `json:"dbUser,omitempty"`
+	SecretManagerARN *string `json:"secretManagerARN,omitempty"`
+	Sql              *string `json:"sql,omitempty"`
```

Comment:
We should also add this to the list of initialisms

Reply:
Same as above then?

Reply:
This field doesn't have the same implications as `awsvpcConfigurations` since it's just one word, but I think it's good to fix it while we're doing the other one.

Reply:
https://github.com/aws-controllers-k8s/pkg/pull/11